framework/console/controllers/FixtureController.php

```diff
@@ -415,13 +415,36 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            $relativeName = $this->getFixtureRelativeName($fixture);
+            $foundFixtures[] = $relativeName;
         }
 
         return $foundFixtures;
     }
 
     /**
+     * Calculates $fixture's name relatively to $templatePath.
+     * Basically, strips getFixturePath() and 'Fixture.php' prefix from fixture's full path
+     * @see getFixturePath()
+     * @param string $fullFixturePath Full fixture path
+     * @return string Relative fixture name
+     */
+    private function getFixtureRelativeName($fullFixturePath)
+    {
+        // $fixturesPath is normalized to unix format in getFixturesPath()
+        $fixturesPath = $this->getFixturePath();
+        // normalize $fixture to unix format
+        $fullFixturePath = str_replace("\\", "/", $fullFixturePath);
```

Comment:
sure, but if the alias resolves to `C:\whatever/fixturepath` it will not match `C:/whatever/fixturepath` as far as I see.